docs/release-notes/1.0.0/release-1.0.0.md

```diff
@@ -0,0 +1,80 @@
+# ML.NET 1.0.0 Release Notes
+
+[ML.NET](https://aka.ms/mlnet) is now 1. :cake:. After 12 preview releases over the last year and implementing many enhancements and API improvements, this release brings you the version `1.0.0` of [ML.NET](https://aka.ms/mlnet). Thank you wholeheartedly for being an awesome comunity. Please continue to use and provide feedback on what should be improved. We will ensure that going forward no breaking API changes are introduced.
```

Comment:
>:cake: [](start = 41, length = 6)

put  the cake in the front or inside the sentence. It's weird to have it form its own sentence.

Reply:
I don't agree with this

Reply:
At least remove the dot after it then. cake is not a sentence

---
In reply to: [279053042](https://github.com/dotnet/machinelearning/pull/3605#discussion_r279053042) [](ancestors = 279053042)

Reply:
removed it alltogether, happy :)